django/templatetags/tz.py

```diff
@@ -7,34 +7,12 @@
 except ImportError:
     from backports import zoneinfo
 
-from django.conf import settings
 from django.template import Library, Node, TemplateSyntaxError
 from django.utils import timezone
 
 register = Library()
 
 
-# RemovedInDjango50Warning: shim to allow catching the exception in the calling
-# scope if pytz is not installed.
-class UnknownTimezoneException(BaseException):
-    pass
-
-
-# RemovedInDjango50Warning
-def timezone_constructor(tzname):
```

Comment:
Same point here... the `zoneinfo` import can be unconditional now I think. 
(Throughout)